btr-api/.env.sample

```diff
@@ -8,6 +8,15 @@ LD_SDK_KEY=
 # Registry Integration Services
 PAY_API_URL=
 PAY_API_VERSION=/api/v1
+LEGAL_API_URL=
+LEGAL_API_VERSION_2=/api/v2
```

Comment:
Will we have to use 2 versions of Legal API here ?
If not naming could be just `LEGAL_API_VERSION`